source/server/backtrace.h

```diff
@@ -67,6 +68,7 @@ class BackwardsTrace : Logger::Loggable<Logger::Id::backtrace> {
    */
   void logTrace() {
     ENVOY_LOG(critical, "Backtrace (use tools/stack_decode.py to get line numbers):");
+    ENVOY_LOG(critical, "Envoy version: {}", VersionInfo::version());
```

Comment:
What do you think of adding EXPECT_LOG_CONTAINS for "Envoy version" in backtrace_test just to make sure we don't lose this?

Reply:
done :)